From 45e4d0b8b5dc0c29424b754e1899ed965cce6574 Mon Sep 17 00:00:00 2001
From: xbw <xbw@rock-chips.com>
Date: Mon, 20 May 2013 11:04:19 +0800
Subject: [PATCH] How to use CONFIG_RK29_SDIO_IRQ_FROM_GPIO for your new 
 wifi-module. For example,please refer to board-rk3168m-tb.c. continue with
 the commit eb1ab60e9f7308d90aff40998178ff2a8a502a29

---
 arch/arm/mach-rk30/board-rk3168m-tb.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/mach-rk30/board-rk3168m-tb.c b/arch/arm/mach-rk30/board-rk3168m-tb.c
index b76afbd07e21..a91d7b9b78d8 100755
--- a/arch/arm/mach-rk30/board-rk3168m-tb.c
+++ b/arch/arm/mach-rk30/board-rk3168m-tb.c
@@ -1007,6 +1007,9 @@ struct rk29_sdmmc_platform_data default_sdmmc1_data = {
 
     #if defined(CONFIG_RK29_SDIO_IRQ_FROM_GPIO)
         .sdio_INT_gpio = RK29SDK_WIFI_SDIO_CARD_INT,
+        #ifdef USE_SDIO_INT_LEVEL
+        .sdio_INT_level = RK30SDK_WIFI_GPIO_WIFI_INT_B_ENABLE_VALUE,
+        #endif
     #endif
 
     .det_pin_info = {    
-- 
2.35.3

